src/main/kotlin/minesweeper/Block.kt

```diff
@@ -0,0 +1,8 @@
+package minesweeper
+
+sealed class Block {
+    abstract val blockText: String
```

Comment:
```suggestion
sealed interface Block {
    val blockText: String
```

sealed interface를 활용해볼 수도 있겠네요 :)

Reply:
interface 로 수정하였습니다.